draft-ietf-tls-tls13.md

```diff
@@ -1905,6 +1948,10 @@ Finite Field Groups (DHE)
 Items in named_group_list are ordered according to the client's
 preferences (most preferred choice first).
 
+If server receives a list that does not include any group acceptable
+to it, it MUST abort the connection by sending "handshake_failure"
```

Comment:
I'd rearrange this to avoid the "it, it". Instead: "does not include any acceptable group, it MUST"